Makefile

```diff
@@ -32,5 +33,17 @@ $(CLEAN_DIRS):
 unit: $(UNITTEST_DIR)/Makefile build-kpatch-build
 	$(MAKE) -C $(UNITTEST_DIR)
 
+integration_slow: $(INTEGRATION_DIR)/Makefile build-kpatch-build build-kpatch build-kmod
+	$(MAKE) -C $(INTEGRATION_DIR) slow
+
+integration_quick: $(INTEGRATION_DIR)/Makefile build-kpatch-build build-kpatch build-kmod
+	$(MAKE) -C $(INTEGRATION_DIR) quick
```

Comment:
> And I'd imagine most people would use the vagrant targets instead.

I assumed people might still want to run it manually on a live system, but I guess vms are much more convenient and we can do without these targets.

Reply:
Looking at this again and there is some value in keeping these because it makes sure the dependencies are met (although at the moment they are all .PHONY but that is something we can improve).

I think they are not making anything worse so there would be no harm in keeping them?

Reply:
They could still cause confusion.  Given the targets' names, if I'm new to the project I could easily assume that I need to run one of these targets to do integration testing.  I don't feel strongly about it though.